src/assets/sql/stored-procedure/getAddressBookingStatisticByConditions.sql

```diff
@@ -0,0 +1,50 @@
+DROP PROCEDURE IF EXISTS `getAddressBookingStatisticByConditions`;
+CREATE PROCEDURE `getAddressBookingStatisticByConditions`(IN `idServiceIntegrated` VARCHAR(200), IN `cityId` INT(11), IN `startDate` VARCHAR(200), IN `endDate` VARCHAR(200), IN `gender` VARCHAR(200), IN `first_time` VARCHAR(200), IN `bhyt` VARCHAR(200))
```

Comment:
bhyt -> healthInsuranceUsed, that is true / false value right?
first_time -> firstTimeBooking, it would be true / false also ?

Reply:
idk why when we pass boolean variable to stored procedure, it convert to "string" even when I declared that input params is Boolean type.  That's why last time when I use Boolean type for 2 both fields, Query run wrongly. That why I changed from BOOLEAN to STRING